### PR TITLE
fix(docs): fixes #632,#633 remove refs to minio and up version

### LINKED ISSUE #632: update armadillo version description in armadillo-setup.sh
Have a look at:
https://github.com/molgenis/molgenis-service-armadillo/blob/7689be392fb7b8bf1922629f3a1266d6304313db/scripts/install/armadillo-setup.sh#L113

Do we need to update the Description to: DataSHIELD Armadillo **4**? Now we have [released](https://github.com/molgenis/molgenis-service-armadillo/releases/tag/v4.1.3) Armadillo 4.1.3?

### PR DESCRIPTION
**Pull request by clemens-tolboom**

### CODE REVIEW

File: docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r [hunk 1]
```diff
@@ -80,7 +79,7 @@ armadillo.upload_table(study, "2_1-core-1_0", subset_core_yearlyrep, "yearlyrep"
 # See if tables are uploaded
 armadillo.list_tables(study)
 
-# Now you can also take a look at the files in the user interface of the MinIO file server. In this case: https://armadillo-minio.test.molgenis.org
+# Now you can also take a look at the files in the user interface of the Armadillo server. In this case: https://armadillo-minio.test.molgenis.org
```
Comment (clemens-tolboom) [inline docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r:82]: There is still a reference to minio

Comment (erikzwart) [reply docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r:82]: To have a look at the files you can use the Armadillo UI projects: https://molgenis.github.io/molgenis-service-armadillo/#/ui?id=project-explorer-
So in this case the url would be https://armadillo.test.molgenis.org/#/projects

**Review by erikzwart — CHANGES_REQUESTED** [review]
If we change the minio url to the Armadillo project url then it will make sense again.

File: docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r (hunk 1 above)
Comment (clemens-tolboom) [reply docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r]: Done

**Pull request merged**